app/_src/deck/use-case.md

```diff
@@ -0,0 +1,103 @@
+---
+title: decK Use Case - Streamlining KongAir APIs
```

Comment:
```suggestion
title: decK Use Case - Streamlining KongAir API Management
```
Does this other title make more sense? Would need to adjust the nav file as well, if that's the case.